src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -91,15 +104,13 @@ Copyright (c) .NET Foundation. All rights reserved.
                  Properties="_CGenExe=$(_Crossgen);
```

Comment:
It's weird that we're passing properties to msbuild invocation that are underscore prefix (signals 'private' by convention only). This seems like a public contract with the RunCrossGenTarget and so its inputs should have good names and not be underscore prefixed.

Also, again avoid obscure abbreviations. There is no point saving 4 characters to say CGen instead of CrossGen.